tests/test_api.py

```diff
@@ -2650,6 +2656,80 @@ def __init__(self, isolated):
             assert isinstance(exc_info.value.cause, ValueError)
             assert "Flatpak build cannot be isolated" in exc_info.value.message
 
+    @pytest.mark.parametrize('isolated', [True, False])
+    def test_operator_csv_modifications_url_cli(self, osbs, isolated):
+        """Test if combination of isolated build and operator_csv_modifications_url
```

Comment:
This wording suggests to me that isolated build with operator csv modification url should raise an exception

Reply:
+1

Reply:
Reworded